test/e2e/get_test_existing_cluster.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2025 Microsoft Corporation
```

Comment:
This test should go to [cluster_get_test.go](https://github.com/Azure/ARO-HCP/blob/main/test/e2e/cluster_get_test.go).

Reply:
Fixed in current commit